1/3.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <fstream>
+
+int main()
+{
+    std::ofstream p;
+    p.open("C:\\Users\\1\\C++\\Model\\f1.txt");
```

Comment:
which file access flags used here? Is file opend for reading or for writing? How could we do this in one line? What if there is an error while opening the file.